release_notes.md

```diff
@@ -5,3 +5,40 @@
 The file describes all the changes, that are affected the usage of the application.
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
+
+[0.1.0] - [26.05.2023]
+
+### New features
+
+`Quicklook`:  Users can check a list of pet sitters available for pet owners. Each row of the list includes an avatar, name + surname, price, short bio and rating of pet sitters.
+
+</details>
+
+<details>
+<summary> CLICK HERE</summary>
+
+<p>
+
+![list of pet sitters](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae)
+
+</p>
+</details>
+
+`Remote check the pet sitter`: App allows users without internet connection tab to the pet sitters' row in the list of sitters and read more detailed information about the sitter:  pet sitter's phone number, rating and full bio.
```

Comment:
For me, it's a confusing phrase `Remote check the pet sitter`. I can check what the pet sitter does somehow 😕 
In the future user can't get the list of pet sitters or check detailed information about it without an internet connection.
The main idea of the detail screen is to show more information than it provides in the card view in the list, that's it.
```suggestion
`Get detailed information about the pet sitter`: The app allows users to get more information on the tap of the pet sitter's row in the list like the pet sitter's phone number, rating, and full bio.
```

Reply:
Agree, thank u. Fixed it.